client/src/pages/Landing/index.js

```diff
@@ -5,6 +5,17 @@ const Landing = () => {
   return (
     <>
       <h1>Landing</h1>
+      <img src="/Landing-Shared.svg"></img>
+      <img src="/github.svg"></img>
```

Comment:
import them using `ReactComponent` e.g. `import { ReactComponent as GithubLogo } from "../../"; ` and use it as a component ` <GithubLogo />`